packages/react-core/README.md

```diff
@@ -45,7 +45,7 @@ You can also find how each component is meant to be used from a design perspecti
 
 
 # Contributing Components
-This library makes use of the babel plugin from [@patternfly/react-styles](../styles/README.md) to enable providing the CSS alongside the components.  This removes the need for consumers to use (style|css|sass)-loaders. For an example of using CSS from core you can reference [Button.js](./src/components/Button.js). For any CSS not provided by core please use the `StyleSheet.create` utility from [@patternfly/react-styles](../styles/README.md). This will prevent collisions with any consumers, and allow the CSS to be bundled with the component.
+This library makes use of the babel plugin from [@patternfly/react-styles](../react-styles/README.md) to enable providing the CSS alongside the components.  This removes the need for consumers to use (style|css|sass)-loaders. For an example of using CSS from core you can reference [Button.js](./src/components/Button/Button.js). For any CSS not provided by core please use the `StyleSheet.create` utility from [@patternfly/react-styles](../react-styles/README.md). This will prevent collisions with any consumers, and allow the CSS to be bundled with the component.
```

Comment:
only the paths were updated here.  They were broken by the new directory layout.

Reply:
At the bottom of the file, `yarn test packages/next/core` should be `yarn test packages/react-core`

Reply:
It was updated as well. https://github.com/patternfly/patternfly-react/pull/404/files/05cc69750511ebecf1b79a3133a2cd54b7f5569b#diff-96fed29cb56decc612348d3253c56876L63